### PR TITLE
Add expander support for NotIn

### PR DESCRIPTION
**Pull request by linsword13**

### CODE REVIEW

File: lib/ramble/ramble/expander.py [hunk 1]
```diff
@@ -799,8 +799,10 @@ def _eval_comparisons(self, node):
         """Handle a comparison node in the ast"""
 
         # Extract In nodes, and call their helper
-        if len(node.ops) == 1 and isinstance(node.ops[0], ast.In):
-            return self._eval_comp_in(node)
+        if len(node.ops) == 1 and (
+            isinstance(node.ops[0], ast.In) or isinstance(node.ops[0], ast.NotIn)
+        ):
+            return self._eval_comp_in_or_not(node, reverse=isinstance(node.ops[0], ast.NotIn))
```
Comment (douglasjacobsen) [inline lib/ramble/ramble/expander.py]: Can the reverse logic be encapsulated in the eval method?

File: lib/ramble/ramble/expander.py [hunk 2]
```diff
@@ -829,8 +831,8 @@ def _eval_comparisons(self, node):
         except KeyError:
             raise SyntaxError("Unsupported binary comparison operator")
 
-    def _eval_comp_in(self, node):
-        """Handle in nodes in the ast
+    def _eval_comp_in_or_not(self, node, reverse):
+        """Handle in and not in nodes in the ast
 
         Perform extraction of `<variable> in <experiment>` syntax.
         Raises an exception if the experiment does not exist.
```
Comment (douglasjacobsen) [inline lib/ramble/ramble/expander.py:839]: Assuming the reverse logic does not move into this method, can you update the doc string to describe the arguments?

**Review by douglasjacobsen — CHANGES_REQUESTED** [review]

File: lib/ramble/ramble/expander.py (hunk 1 above)
Comment (linsword13) [reply lib/ramble/ramble/expander.py]: Oh right indeed, that'd be much simpler. Updated.

File: lib/ramble/ramble/expander.py (hunk 2 above)
Comment (linsword13) [reply lib/ramble/ramble/expander.py:839]: Moved the logic up.

**Review by douglasjacobsen — APPROVED** [review]

**Pull request merged**